mmv1/products/apphub/Application.yaml

```diff
@@ -0,0 +1,199 @@
+# Copyright 2024 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+base_url: projects/{{project}}/locations/{{location}}/applications
+create_url: projects/{{project}}/locations/{{location}}/applications?applicationId={{application_id}}
+self_link: projects/{{project}}/locations/{{location}}/applications/{{application_id}}
+id_format: projects/{{project}}/locations/{{location}}/applications/{{application_id}}
+import_format:
+  - projects/{{project}}/locations/{{location}}/applications/{{application_id}}
+name: Application
+description: Description
+autogen_async: true
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: "application_basic"
+    primary_resource_id: "example"
+    config_path: "templates/terraform/examples/apphub_application_basic.tf.erb"
+    skip_test: true
```

Comment:
Why skip test?

Reply:
The generated tests add random string value even to `Enum` fields when taken as example. That's why removed them from `vars`. But we need users to provide `location` and `scope_type` , hence adding those to documentation and skipping tests for that.

Reply:
You can just hard-code the enum into the test rather than passing it through, then you won't need to skip test

Reply:
Got it. Fixed.